disquaire_project/manager/models.py

```diff
@@ -20,11 +20,13 @@ def __str__(self):
         return self.name
 
 class Disk(models.Model):
+    # There is no unicity constraint and no index on the reference. Are you sure you need this?
     reference = models.IntegerField('référence')
     created_at = models.DateTimeField('Date de création', auto_now_add=True)
+    # what does "active" mean?
```

Comment:
It means it should appear on the front office as "available". A user can book an activated disk only.